.github/workflows/deploy.yml

```
@@ -0,0 +1,23 @@
+name: Deploy to EC2 via Tailscale
+on:
+    push:
+        branches:
+        - main
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 12 but found 8

(indentation)